### PR TITLE
Support filter param for histograms API request

### PR DESCRIPTION
**Pull request by dariashka**

### CODE REVIEW

File: src/resources/experimental/histograms-resource.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 export default function HistogramsResource({apiHandler}) {
     return {
-        getTransactionHistogramReport({periodStart = '', periodEnd = '', aggregationField = '', aggregationPeriod = '', metric = '', criteria = null} = {}) {
+        getTransactionHistogramReport({periodStart = '', periodEnd = '', aggregationField = '', aggregationPeriod = '', metric = '', filter = '', criteria = null} = {}) {
```
Comment (mclassic) [inline src/resources/experimental/histograms-resource.js:3]:
```suggestion
        getTransactionHistogramReport({periodStart = '', periodEnd = '', aggregationField = '', aggregationPeriod = '', metric = '', filter = null, criteria = null} = {}) {
```

Comment (adamaltman) [reply src/resources/experimental/histograms-resource.js:3]: I wonder why this method has **criteria** at all?

Comment (mclassic) [reply src/resources/experimental/histograms-resource.js:3]: The report can parse criteria, but you're right, it shouldn't, so maybe you should remove that @dariashka (the `criteria`)

**Comment by mclassic** [comment]
We'll have to bump the version from 18.0.0 to 19.0.0 because of backwards-incompatible API change if you are planning to release this for immediate use.

**Pull request merged**